src/schema/imported/devtools.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "devtools",
   "permissions": [
-    "devtools"
+    "manifest:devtools_page"
```

Comment:
I'm going to ask aswan about this. We might want to overwrite this change, it looks like internally it ends up adding the devtools pref but I don't understand what the deal with this is.

Reply:
I discussed this on IRC and this basically means including the "devtools_page" manifest property with implicitly request the "devtools" permission. I tested an extension requesting these permissions and they are working as expected.